SimpleWAWS/Code/CsmExtensions/CsmManager.cs

```diff
@@ -159,7 +172,12 @@ static CsmManager()
             await response.EnsureSuccessStatusCodeWithFullError();
 
             var result = await response.Content.ReadAsAsync<CsmSubscriptionsArray>();
-            return result.value.ToDictionary(k => k.displayName, v => v.subscriptionId);
+
+            response = await jenkinsClient.HttpInvoke(HttpMethod.Get, ArmUriTemplates.Subscriptions.Bind(""));
+            await response.EnsureSuccessStatusCodeWithFullError();
+
+            var result2 = await response.Content.ReadAsAsync<CsmSubscriptionsArray>();
+            return result.value.Union(result2.value).ToDictionary(k => k.displayName, v => v.subscriptionId);
```

Comment:
Yes. For this to work, we'll need to have the subscriptionId for Jenkins subscriptions in the environment variable instead of subscriptionNames. I've noted a comment [here](https://github.com/fashaikh/SimpleWAWS-1/blob/dev/SimpleWAWS/Code/ResourcesManager.cs#L74L75)